frontend/src/views/PrivacyPolicy.vue

```diff
@@ -0,0 +1,156 @@
+    
+      When {{ productName }} projects are either completed or deleted, aggregated, anonymous data including the number of documents, 
+      number of annotators, number of annotations are sent to the developers of {{ productName }}, GATE. {{ productName }} is provided as free, 
+      open source software and collecting usage statistics allows GATE to demonstrate its usage to funders, 
+      allowing us to support its development and maintenance.
+    </p>
+
+    <p>
+      <h2>How we store personal information</h2>
+
+      Your information is securely stored.
+
+      Data is kept within a {{ productName }} instance as long as the instance is running; or until it is deleted. 
+      Once an instance is no longer running, data will be kept for a duration according to the agreement between the host and administrator.
+    </p>
+
+    <p>
+      <h2>Your data protection rights</h2>
+
+      Under data protection law, you have rights including:<br>
+      
+      <strong>Your right of access</strong> - You have the right to ask the host or administrator(s) for copies of your personal information.<br>
+
+      <strong>Your right to rectification</strong> - You have the right to ask the host or administrator(s) to rectify personal information you think is inaccurate. 
+      You also have the right to ask the host or administrator(s) to complete information you think is incomplete.<br>
+
+      <strong>Your right to erasure</strong> - You have the right to the host or administrator(s) to erase your personal information in certain circumstances. 
+      Please use the facilities within {{ productName }} to delete your account if required, at which point any annotations or documents will remain, but your account will be anonymised.<br>
+
+      <strong>Your right to restriction of processing</strong> - You have the right to ask us to restrict the processing of your personal information in certain circumstances.<br>
+
+      <strong>Your right to object to processing</strong> - You have the right to object to the processing of your personal information in certain circumstances.<br>
+
+      <strong>Your right to data portability</strong> - You have the right to ask that we transfer the personal information you gave us to another organisation, or to you, in certain circumstances.<br>
+      <br>
+      You are not required to pay any charge for exercising your rights. If you make a request, we have one month to respond to you.<br>
+      
+      Please contact the host and/or administrators at the contact details above if you wish to make a request.<br>
+
+    </p>
+
+    <p>
+      <h2>How to complain</h2>
+      If you have any concerns about the use of your personal information in {{ productName }}, 
+      you can make a complaint to the host and or administrator at the contact details above.
+      You can also complain to the Information Commissioner’s Office (ICO) if you are unhappy with how your data is used.
```

Comment:
This is UK specific but I guess anyone outside the UK will be providing a whole custom policy in their native language anyway.